code/go/0chain.net/go.mod

```diff
@@ -1,9 +1,10 @@
 module 0chain.net
 
-go 1.15
+go 1.16
 
 require (
 	github.com/0chain/gorocksdb v0.0.0-20181010114359-8752a9433481
+	github.com/alicebob/miniredis/v2 v2.14.3
 	github.com/asaskevich/govalidator v0.0.0-20190424111038-f61b66f89f4a
```

Comment:
Why are changes to go.mod needed? If this is not part of the issue being fixed these changes should be removed.

Reply:
Pls see https://golang.org/ref/mod#go-mod-tidy

Reply:
I suppose it will be good practice to execute the `go mod tidy` command before push a commit to the remote repository.

Reply:
I am sure `go mod tidy` is working fine. ​But this is unrelated to your changes, so the `go.mod`changes are due to other files in your local source. Even if your source files are not dirty, there is no way to confirm this.
Try to avoid unrelated refactoring, there are too many ways this can go wrong. 
If you are worried get a clean copy of master down and run `go mod tidy` on that. You can raise an issue and create a Pr if changes are suggested.